sdk/cosmos/azure-cosmos/azure/cosmos/aio/_database.py

```diff
@@ -324,7 +324,8 @@ async def create_container_if_not_exists(
                 unique_key_policy=unique_key_policy,
                 conflict_resolution_policy=conflict_resolution_policy,
                 analytical_storage_ttl=analytical_storage_ttl,
-                computed_properties=computed_properties
+                computed_properties=computed_properties,
+                **kwargs
```

Comment:
Looks like we might need a few more tweaks here - currently we also have these kwargs documented for this method:
- session_token
- initial_headers
- etag
- match_condition

The latter two can't be used with the `read` method, so probably need to be popped at the beginning and only passed into the `create_container` method.
The former two can technically be passed into both the `read` and `create_container` methods - would this be the expected behaviour? Session token is probably fine, but not sure about initial_headers....

Reply:
I really wish we had never included `initial_headers` anywhere, but yeah all of those should be relevant to just about any method - pretty sure they get used in the `build_options` method, we might just be missing the documentation on them. I can go ahead and add that.